### PR TITLE
improved accessibility in form labelling #4079

### PR DESCRIPTION
**Pull request by amycruz97**
**This pull request makes the following changes:**

* Fixes [ushadi/platform#4079](https://github.com/ushahidi/platform/issues/4079)

**Issues detected:**
* some forms have missing labels and for attribute

Testing checklist:
- [x]  Login to your account.
- [x]  Navigate to the forms there is a visible, meaningful label describing what should be entered into the form field
- [x] I certify that I ran my checklist

Fixes ushahidi/platform#4079 .

Ping @ushahidi/platform

### CODE REVIEW

**Comment by amycruz97** [comment]
how can i sign the contributor agreements

**Comment by rowasc** [comment]
@amycruz97 try https://docs.google.com/forms/d/15LyeKTOP36T5u3290o4hsmqv79v-m_s2QSmgMJXpYp8/viewform here 
Then run "[clabot:check]" (without the quotation marks) so the bot knows.

**Comment by amycruz97** [comment]
> @amycruz97 try https://docs.google.com/forms/d/15LyeKTOP36T5u3290o4hsmqv79v-m_s2QSmgMJXpYp8/viewform here
> Then run "[clabot:check]" (without the quotation marks) so the bot knows.

run [clabot:check] where ..in my local terminal or where

**Comment by amycruz97** [comment]
[clabot:check]

**Comment by amycruz97** [comment]
clabot:check

**Comment by amycruz97** [comment]
[clabot:check]

**Comment by Angamanga** [comment]
@amycruz97 Could you add a test-checklist for this please?

**Comment by amycruz97** [comment]
teat checklist how do i do that

**Comment by Angamanga** [comment]
@amycruz97 A test-checklist is a list you add to the description of the PR and that works as a guide when doing review and QA. You can see an example here: https://github.com/ushahidi/platform-client/pull/1564

**Comment by amycruz97** [comment]
[clabot:check]

**Comment by amycruz97** [comment]
[clabot:check]

**Comment by amycruz97** [comment]
[clabot:check]

**Comment by amycruz97** [comment]
> @amycruz97 A test-checklist is a list you add to the description of the PR and that works as a guide when doing review and QA. You can see an example here: #1564

yh i added a test checklist thanks

**Comment by amycruz97** [comment]
@rowasc pls check this pull request

File: app/common/directives/file-upload.html [hunk 1]
```diff
@@ -1 +1,3 @@
+<label>
```
Comment (rowasc) [inline app/common/directives/file-upload.html:1]: The label text is missing here

File: app/common/directives/filter-system/filter-searchbar.html [hunk 2]
```diff
@@ -3,6 +3,7 @@
     <div class="form-field">
         <label class="hidden" translate="toolbar.searchbar.search_entity">Search</label>
         <div class="input-with-icon">
+            <label>
```
Comment (rowasc) [inline app/common/directives/filter-system/filter-searchbar.html:6]: The label text is missing here

File: app/main/activity/activity-timeline.html [hunk 3]
```diff
@@ -6,7 +6,9 @@
        <article class="postcard">
            <div class="post-band" ng-style="{backgroundColor: form.color}"></div>
            <div class="listing-item-select">
+               <label>
```
Comment (rowasc) [inline app/main/activity/activity-timeline.html:9]: The label text is missing here

**Review by rowasc — CHANGES_REQUESTED** [review]
Hey @amycruz97 this is a good start ! Thanks
A couple of things
- a lot of labels are simply missing the text description on them (as shown here https://github.com/ushahidi/platform/issues/4079) 
- please use hyphen instead of underscore in the IDs 
Thank you !

**Comment by amycruz97** [comment]
> Hey @amycruz97 this is a good start ! Thanks
> A couple of things
> 
> * a lot of labels are simply missing the text description on them (as shown here [ushahidi/platform#4079](https://github.com/ushahidi/platform/issues/4079))
> * please use hyphen instead of underscore in the IDs
>   Please note I only added specific comments in a few instances but the label text are missing in others as well.
>   Thank you !

Thanks on it

**Comment by amycruz97** [comment]
@rowasc pls i have made the changes, kindly review the pull request Thanks

**Comment by amycruz97** [comment]
@Angamanga @rowasc pls review the pull request

**Comment by Angamanga** [comment]
Hi @amycruz97! This looks fine now, thank you for your contribution. As you see there are a lot of conflicts in the files, it means that these issues was already fixed by other PRs and I will close this. It doesn't mean we don't value your work and it is still counted for in the Outreachy evaluation, its just that we had so many contributors this year working on the same issues. Thank you again for your hard work 🎉 .

**Pull request closed without merging**